pkg/master/master.go

```diff
@@ -263,7 +268,7 @@ func (m *Master) InstallAPIs(c *Config) {
 		autoscalingGroupMeta := registered.GroupOrDie(autoscaling.GroupName)
 
 		// Hard code preferred group version to autoscaling/v1
-		autoscalingGroupMeta.GroupVersion = unversioned.GroupVersion{Group: "autoscaling", Version: "v1"}
+		autoscalingGroupMeta.GroupVersion = autoscalingGroupVersion
```

Comment:
Undo this change-- the autoscaling group versions should not be connected to the version of the sub resources.